phpcs.xml.dist

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<ruleset name="WordPress coding standards">
+	<config name="ignore_warnings_on_exit" value="1" /><!-- Ignore warnings for now. -->
+
+	<arg name="basepath" value="." /><!-- Show file paths relative to the project root. -->
+	<arg name="extensions" value="php" />
+	<arg name="colors" />
+	<arg value="s" /><!-- Show sniff codes in all reports. -->
+
+	<!-- Includes WordPress-Core, WordPress-Docs and WordPress-Extra rulesets. -->
+	<rule ref="WordPress">
+		<type>warning</type><!-- TODO Remove this after fixing all warnings. -->
```

Comment:
All WP core standard checks are being reported as warnings until we'll fix them. Can be done slowly as we work on features.